Sources/DTTJailbreakDetection/DTTJailbreakDetection.m

```diff
@@ -20,7 +20,7 @@
 //  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-#import "DTTJailbreakDetection.h"
+#import "include/DTTJailbreakDetection.h"
```

Comment:
Please add `include` to the header search paths instead.

Reply:
Hello @thii 
It works even without any header search paths changes, tested with Carthage 0.34.0.